src/styles.scss

```diff
@@ -30,3 +30,7 @@ ul {
 	overflow: hidden;
 	padding: 0 2rem;
 }
+
+button {
```

Comment:
plese try to avoid using tag selectors

Reply:
Sure

Reply:
it's only in style.scss which one in root, i did this for giving the same behavior for all buttons or for all ul's in project, how can i use here class